### ISSUE TITLE
Speed up generation of notable release notes

### ISSUE THREAD

**Issue by szczepiq** [mockito/mockito-release-tools#58]
I've ported current version of release tools to Mockito, PR is open: https://github.com/mockito/mockito/pull/1018 I noticed that the release is slow due to getting contributors from GitHub. When I commented out the code that gets the contributor/GH id mapping the build takes 10secs instead of over a minute.

Suggested plan:
- [x] - disable getting contributors from the notable release notes (notable notes don't use contributors) - #59
- [ ] - try to address the issue so that the build is fast

**Comment by mstachniuk** [comment]
Logs from this build are unavailable due to Travis Database outage. You can follow a status here: https://www.traviscistatus.com/incidents/93wfcv0ksqrz

**Comment by mstachniuk** [comment]
@szczepiq  I tried to check locally whats happens. Running `./gradlew updateNotableReleaseNotes` on `sf` branch I got an error: `fatal: ambiguous argument 'v2.7.0': unknown revision or path not in the working tree.`
I checked and this tag doesn't exist in Mockito repo.

Regarding long gets contributors from GH, are you sure is it slow? I checked locally `previewReleaseNotes` for revisions: v2.3.11 -> HEAD and it took 13 seconds. I checked in debug mode it was 9 requests to GH.
This same for revisions: v1.9.0 ->  takes 48 seconds and 30 requests to GH. It's a lot because of deleted accounts on GH.

How should looks notable notes? Should they create links do contributors profile?
